### ISSUE TITLE
Offboard Mateo

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#10159]
## IT Permissions and Access

- [ ] 1password
- [ ] Slack
- [ ] Miro
- [ ] Vision Zero
- [ ] Moped
- [ ] Knack Applications
- [ ] DTS Portal
- [ ] Data Tracker
- [ ] Signs and Markings
- [ ] GitHub - (reassign issues, too)
- [ ] Google Drive
~~- [ ] SERF~~

## Admin

- [ ] DTS email DL
- [ ] DTS email distribution
- [ ] About Us
- [ ] DTS Team Slide
- [ ] DTS SharePoint team lists - references the "About Us" page on austinmobility.io

## Recover organization assets

- [ ] Badge - Remove access
- [ ] Laptop - check DTS hardware closet
- [ ] Docking Station
- [ ] Monitors
- [ ] Desk Riser - at Cameron
- [ ] Chair - at Cameron

**Comment by mateoclarke** [comment]
Here's the immediate action items we discussed Monday:

- [ ] Github: Make a PR to the README to replace Mateo with Frank as 1 of 2 ATD representatives and org Owners for the City of Austin GIthub Governance group. @mateoclarke 
- [ ] CRIS: Chia and Frank to create personal CRIS accounts (both Search & Extract) and update credentials in ETLs as necessary @frankhereford @chiaberry 
- [ ] AWS: Assign someone to audit IAM account credentials & Enable MFA for root user (if possible using 1PW’s MFA service?) @chiaberry 
- [ ] Slack: Ask for Diana to replace Mateo as workspace admin @mateoclarke 
- [ ] GoDaddy: Migrate all GoDaddy domains to AWS Route 53 @frankhereford 
- [ ] GoDaddy: Check in with Scott R to ask if he controls the Mastercard on the GoDaddy account @frankhereford 
- [ ] Heroku: Frank to create an issue to RIP Heroku Test - Phase out existing Dynos and remove pointer to Heroku on moped-test subdomain DNS. Remove GH Action integrations out of Moped repo. @frankhereford 
- [ ] Nearmap: Contact CTM to confirm who has access to the “Austin_Transportion” API key. Ask about multiple keys by application @frankhereford @chiaberry 
- [ ] Netlify: look at pricing for Pro account and propose any changes to team @johnclary 
- [ ] 1Password: Remove Mateo from Owners/Admins @dianamartin @amenity 

I also think it wouldn't be a bad idea to deactivate my accounts for things that have a monthly cost or security concerns. The most important thing to keep is my COA email because it would be the most effort to re-provision if/when I come back as a contractor. The rest I leave up to y'alls discretion.

**Comment by mateoclarke** [comment]
Here's the PR for Github Governance. https://github.com/cityofaustin/coa-github-governance/pull/3

I would give it a week for any of the other point of contacts to approve but I wouldn't necessarily let lack of approval block action. Once this PR is merged, @amenity or I can update the ownership permissions within the Github org

**Comment by amenity** [comment]
@johnclary @chiaberry @dianamartin - I've chipped away at this a little and @patrickm02L unassigned Mateo from VZ issues. Can y'all help me
- Identify which issues should go into the "Wait" list
- Do/delegate the others as time permits

**Comment by dianamartin** [comment]
I still have to deactivate this badge. I think I'll be able to do it tomorrow 9/21

**Comment by frankhereford** [comment]
still chipping away; today I got:

* ✅ ec2 instances
* ✅ aws
* ✅ on-prem servers
* ✅ Moped and VZ DB + RR
* ✅ GitHub team

**Comment by chiaberry** [comment]
moved the GoDaddy todos into its own issue https://github.com/cityofaustin/atd-data-tech/issues/10790

**Comment by amenity** [comment]
Verrrry close here! @dianamartin - Any idea what is meant by `DTS SharePoint team lists - references the "About Us" page on austinmobility.io`?

**Comment by dianamartin** [comment]
@amenity he's already removed from the DTS distribution list, I think the emergency escalation matrix that Karo was working on updating does need to be updated.

**Comment by chiaberry** [comment]
Sharepoint work is captured in a separate issue, so closing this one!
